docs/slides/population_structure/index.qmd

```diff
@@ -782,45 +721,116 @@ path <- file.path(getwd(), "data")
 #| fig-height: 6
 #| tidy: true
 #| tidy.opts: { width.cutoff: 78 }
-df <- read.delim(file.path(path, "out_abbababa_results.txt"), header = T, sep = "\t")
-df <- df[grepl("Neandertal", as.character(df$H3)), ]
-df <- df[grepl("CEU", as.character(df$H2)), ]
-df <- df[grepl("YRI", as.character(df$H1)), ]
-YRI <- matrix(unlist(strsplit(as.character(df$H1), "\\.")), ncol = 9, byrow = T)
-df$H1 <- paste(YRI[, 6], YRI[, 2], sep = "_")
-CEU <- matrix(unlist(strsplit(as.character(df$H2), "\\.")), ncol = 9, byrow = T)
-df$H2 <- paste(CEU[, 6], CEU[, 2], sep = "_")
-df$H3 <- "NEAND"
-df[1:8, 1:8]
+
+df<-read.delim(paste0(path,"out_abbababa_results.txt"),header=T,sep="\t")
+df<-df[grepl("Neandertal",as.character(df$H3)),]
+df<-df[grepl("CEU",as.character(df$H2)),]
+df<-df[grepl("YRI",as.character(df$H1)),]
+YRI<-matrix(unlist(strsplit(as.character(df$H1),"\\.")),ncol=9,byrow=T)
+df$H1<-paste(YRI[,6], YRI[,2], sep="_")
+CEU<-matrix(unlist(strsplit(as.character(df$H2),"\\.")),ncol=9,byrow=T)
+df$H2<-paste(CEU[,6], CEU[,2], sep="_")
+df$H3<-"NEAND"
+df[1:8,1:8]
+```
+:::
+::: {.column width="50%"}
+
+```{r}
+#| eval: true
+#| echo: false
+path<-"/home/nikolay/Documents/Teaching/PopGen_2023/data/"
+df<-read.delim(paste0(path,"out_abbababa_results.txt"),header=T,sep="\t")
+df<-df[grepl("Neandertal",as.character(df$H3)),]
+df<-df[grepl("CEU",as.character(df$H2)),]
+df<-df[grepl("YRI",as.character(df$H1)),]
 ```
 
+```{r}
+#| class-source: my_code_size_abitsmaller
+#| classes: my_code_size_abitsmaller
+#| fig-width: 10
+#| fig-height: 6
+#| tidy: true
+#| tidy.opts: { width.cutoff: 78 }
+
+boxplot(df$nABBA,df$nBABA,names=c("ABBA","BABA"),ylab="COUNTS",col=2)
+wilcox.test(df$nABBA,df$nBABA)
+```
 :::
+::::
+
+
+## D-statistic computation for higher coverage samples
+
+:::: {.columns}
 ::: {.column width="50%"}
+D-statistic analysis requires large number of genomic sites, therefore the low-coverage 1000G data are not optimal. Now, we are going to use French and Yoruba samples from the **Draft Neanderthal genome** project where Nenandertal introgression was first detected:
+
+```{bash}
+#| eval: false
+#| class-source: my_code_size_abitsmaller
+#| classes: my_code_size_abitsmaller
+#| fig-width: 10
+#| fig-height: 6
+#| tidy: true
+#| tidy.opts: { width.cutoff: 78 }
+
+echo Neandertal.bam > bams
+echo Yoruba.bam >> bams
+echo French.bam >> bams
+angsd -out out -doAbbababa 1 -bam bams -doCounts 1 -anc anc.fa
+Rscript jackKnife.R file=out.abbababa indNames=bams outfile=out_abbababa_highcov
+```
 
 ```{r}
 #| eval: true
 #| echo: false
-path <- file.path(getwd(), "data")
-df <- read.delim(file.path(path, "out_abbababa_results.txt"), header = T, sep = "\t")
-df <- df[grepl("Neandertal", as.character(df$H3)), ]
-df <- df[grepl("CEU", as.character(df$H2)), ]
-df <- df[grepl("YRI", as.character(df$H1)), ]
+path<-"/home/nikolay/Documents/Teaching/PopGen_2023/data/"
```

Comment:
Ditto. If you want to include example data, do so in the sub-directory `data` relative to the directory `docs/slides/population_structure`. Keep the dataset small though.